python/core/auto_generated/geometry/qgsrectangle.sip.in

```diff
@@ -111,10 +111,18 @@ Set the minimum y value.
 Set the maximum y value.
 %End
 
-    void setMinimal() /HoldGIL/;
+    void setNull() /HoldGIL/;
+%Docstring
+Mark a rectangle as being null.
```

Comment:
Might be worth giving more details here about what null means (like `setMinimal` has).

Reply:
I've force pushed a commit adding more documentation and also the missing \since indication